kawin/tests/test_thermodynamics.py

```diff
@@ -239,6 +240,19 @@ def test_IC_ternary_output():
     assert caarray.shape == (3, 2)
     assert cbarray.shape == (3, 2)
 
+
+def test_initialize_with_pycalphad_database():
```

Comment:
You could add an additional test to check that the databases and phases are stored correctly. This can also be done by checking an output of a function for a Thermodynamics object that was initialized with different database or phase parameters.

Reply:
Wouldn't the initialization of the GeneralThermdynamics class fail if they weren't being stored correctly? Both the database and phase list are used to build phase records, which I think wouldn't work if there were problems. Or am I misunderstanding your comment?

Reply:
You're correct on this. I forgot that the database and phase list are used upon initialization. This test is fine as is then.